eng/templates/jobs/e2e-tests.yml

```diff
@@ -0,0 +1,127 @@
+parameters:
+- name: isProd
+  type: boolean
+- name: name
+  type: string
+- name: displayName
+  type: string
+- name: testFilter
+  type: string
+
+jobs:
+- job: ${{ parameters.name }}
+  displayName: ${{ parameters.displayName }}
+  timeoutInMinutes: 120
```

Comment:
Do we maybe want to have this as a parameter and have a lower default? From the runs it seems 60 minutes per test category should be enough

Reply:
Good catch, 60 should be enough